src/massaction_jump_utils.jl

```diff
@@ -59,10 +59,17 @@ function make_majump(rs, specmap, ratemap, params)
         error("Empty net stoichiometry vectors for mass action reactions are not allowed.")
     end
 
+    # populate dummy context with params as local variables
```

Comment:
Thanks for the help! Since populating the dummy module only needs to be done once, would it not make sense to move lines 62-66 to a higher level in the call tree? (Even just to `network_to_jumpset`). You could then pass the module into `make_majump`. Also, in this case maybe use a more descriptive name for the parameter context module input variable.